test/prefer-includes.js

```diff
@@ -0,0 +1,69 @@
+import test from 'ava';
+import avaRuleTester from 'eslint-ava-rule-tester';
+import rule from '../rules/prefer-includes';
+
+const ruleTester = avaRuleTester(test, {
+	env: {
+		es6: true
+	}
+});
+
+const errors = [{
+	ruleId: 'prefer-includes',
+	message: 'Use `.includes()`, not .indexOf(), when checking for existence.'
+}];
+
+ruleTester.run('prefer-includes', rule, {
+	valid: [
+		'str.indexOf(\'foo\') !== -n',
+		'str.indexOf(\'foo\') !== 1',
+		'!str.indexOf(\'foo\') === 1',
+		'!str.indexOf(\'foo\') === -n',
+		'str.includes(\'foo\')',
+		'\'foobar\'.includes(\'foo\')',
+		'[1,2,3].includes(4)',
+		'"str.indexOf(\'foo\') !== -1"'
```

Comment:
Could you add, just to be sure that nothing odds is happening, the following tests:
```js
`a + b` // you target binary expressions. If your checks have gaps, code like this could crash the plugin or create false positives
`indexOf !== -1`
// And whatever else you feel like adding :D
```